patches/tModLoader/Terraria.Localization.Content/zh-Hans.tModLoader.json

```diff
@@ -334,23 +334,12 @@
 
 		// Misc
 		"OK": "好的",
+		// "NoProblemsFound": "No Problems Found. If a problem occurs it will be shown here",
 		"Continue": "继续",
 		"PatreonSetTooltip": "感谢您的支持!",
 		"WorldGenError": "生成世界时遇到问题",
 		"WorldIODataException": "游戏遇到一个关于地图的模组数据问题.",
 		"ServerCrash": "服务器崩溃: {0}\n{1}\n\n请检查[server.log]以确定是否由模组导致,否则请转到tModLoader Discord中的#support"
-

```

Comment:
You removed this commit:  https://github.com/tModLoader/tModLoader/commit/4356f94b797da276065c43b423c449fa5add310f
You'll have to put it back

Reply:
ended up deleting the file, then downloading from Github and placing it back edited, lol